setup/controllers/language.php

```diff
@@ -7,12 +7,15 @@
  */
 /* parse language selection */
 if (!empty($_POST['proceed'])) {
-    $language= 'en';
-    $cookiePath = preg_replace('#[/\\\\]$#', '', dirname(dirname($_SERVER['REQUEST_URI'])));
```

Comment:
The cookie path is still being used below on line 17 (15), which is now undefined.